app/models/folio/item.rb

```diff
@@ -123,6 +127,7 @@ def self.from_hash(dyn)
                        dyn['chronology']].filter_map(&:presence).join(' '),
           public_note: dyn.fetch('notes').find { |note| note.dig('itemNoteType', 'name') == 'Public' }&.fetch('note'),
           effective_location: Location.from_hash(dyn.fetch('effectiveLocation')),
+          permanent_location_code: dyn.fetch('permanentLocation').fetch('code'),
```

Comment:
We also need to fall back on the holding's effective Location; we're no longer guaranteed an item-level permanent location.

Reply:
I believe the fallback is complete.